sql/2021/pwa/pwa_blink_usage.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# PWA features tracked in blink_features.usage
+SELECT
+  client,
+  feature,
+  num_urls,
+  total_urls
+FROM
+  `httparchive.blink_features.usage`
+WHERE
+  yyyymmdd = '20210701' AND
+  (
+    feature LIKE '%ServiceWorker%' OR
+    feature LIKE '%BackgroundSync%' OR
+    feature LIKE '%GetInstalledRelatedApps%'
+  )
+ORDER BY
+  client,
+  num_urls,
+  feature,
```

Comment:
BigQuery console complaints that there's a comma at the end of this query, but I see that it's also present in the [2020 version](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/pwa/pwa_blink_usage.sql).

Reply:
Removed.